src/structures/ClientUser.js

```diff
@@ -16,17 +16,21 @@ class ClientUser extends Structures.get('User') {
   _patch(data) {
     super._patch(data);
 
-    /**
-     * Whether or not this account has been verified
-     * @type {boolean}
-     */
-    this.verified = data.verified;
-
-    /**
-     * If the bot's {@link ClientApplication#owner Owner} has MFA enabled on their account
-     * @type {?boolean}
-     */
-    this.mfaEnabled = typeof data.mfa_enabled === 'boolean' ? data.mfa_enabled : null;
+    if ('verified' in data) {
+      /**
+       * Whether or not this account has been verified
+       * @type {boolean}
+       */
+      this.verified = data.verified;
+    }
+
+    if ('mfa_enabled' in data) {
+      /**
+       * If the bot's {@link ClientApplication#owner Owner} has MFA enabled on their account
+       * @type {?boolean}
+       */
+      this.mfaEnabled = typeof data.mfa_enabled === 'boolean' ? data.mfa_enabled : null;
```

Comment:
It looks like the original behaviour here was to set `mfaEnabled` to null if the property was not defined in data. With this change, `mfaEnabled` will only be set to null if it is defined in the data.

Could you make a change that sets `mfaEnabled` to null if it is currently undefined?

Reply:
Just to clarify - if `this.mfaEnabled` is currently undefined, rather than `data.mfaEnabled`?

Reply:
Yes, if `this.mfaEnabled` is undefined it should be set to null